src/test/java/com/shop/frontend/stepdefinitions/Stepdefs.java

```diff
@@ -103,6 +104,28 @@ public void heading_should_be(String headingText) {
         assertEquals(headingText, mainPage.getHeadingText(), "Heading is not correct.");
     }
 
+    // Author: Jim
+    @Given("User is still on main page")
+    public void user_is_still_on_main_page() {
```

Comment:
To me, the phrase "user_is_still_on_main_page" implies that we are verifying that the user is on the main page. However, the stepdef simply navigates to the mainpage. A suggestion is to change this to "user is on main page" or "user navigates to the main page"

Reply:
Thanks, i get what you mean, to me it felt ok as it is a given and in the context of the cucumber feature as a narrative. But all the possibilities work fine for me. :)